packages/gatsby/src/redux/nodes.js

```diff
@@ -103,6 +103,34 @@ exports.saveResolvedNodes = async (nodeTypeNames, resolver) => {
   }
 }
 
+/**
+ * Get node and save path dependency.
+ *
+ * @param {string} typeName
+ * @param {string} id
+ * @returns {Object|void} node
+ */
+const getResolvedNode = (typeName, id) => {
+  const { nodesByType, resolvedNodesCache } = store.getState()
+  const nodes /*: Map<mixed> */ = nodesByType.get(typeName)
+
+  if (!nodes) return undefined
```

Comment:
**nitpick**
why not use null? I also think we opt for brackets
```
if (!nodes) {
  return undefined
}
```

Reply:
I've lately swayed from using `null` as an object default to `undefined` for all the defaults. I don't mind changing it to `null`.

I'll add the brackets. We can have linting autofix that.

Reply:
I was just wondering, that's all :) I wonder if null is better for v8 hot code path :shrug: